drivers/gles3/rasterizer_storage_gles3.cpp

```diff
@@ -7233,6 +7253,30 @@ void RasterizerStorageGLES3::_render_target_allocate(RenderTarget *rt) {
 
 		glFramebufferRenderbuffer(GL_FRAMEBUFFER, GL_COLOR_ATTACHMENT0, GL_RENDERBUFFER, rt->buffers.diffuse);
 
+		GLuint diffuse;
+		glGenTextures(1, &diffuse);
```

Comment:
This goes the same for the others below:

You should only be allocating space for textures when forcing mrts. Otherwise you are doubling the memory cost for all users for a feature that they likely aren't using.